src/main.ts

```diff
@@ -1,14 +1,26 @@
 import { NestFactory } from '@nestjs/core';
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 import { ValidationPipe } from '@nestjs/common';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
   app.useGlobalPipes(
     new ValidationPipe({
       whitelist: true,
     }),
   );
+
+  const config = new DocumentBuilder()
+    .setTitle('Recipe App')
+    .setDescription('The recipe app API description')
+    .setVersion('1.0')
+    .addTag('Recipes')
+    .build();
+  const document = SwaggerModule.createDocument(app, config);
+  SwaggerModule.setup('api', app, document);
+
```

Comment:
consider moving the swagger configuration to a separate config file, import this file here and call some method like `initializeSwagger()` or `configureSwagger()` or something similar